term.c

```diff
@@ -1440,6 +1444,105 @@ term_lower_dtr(int fd)
 /***************************************************************************/
 
 int
+term_raise_rts(int fd)
+{
+	int rval, r, i;
+
+	rval = 0;
+
+	do { /* dummy */
+
+		i = term_find(fd);
+		if ( i < 0 ) {
+			rval = -1;
+			break;
+		}
+
+#ifdef __linux__
+		{
+			int opins = TIOCM_RTS;
+
+			r = ioctl(fd, TIOCMBIS, &opins);
+			if ( r < 0 ) {
+				term_errno = TERM_ERTSUP;
+				rval = -1;
+				break;
+			}
+		}
+#else
+		r = tcsetattr(fd, TCSANOW, &term.currtermios[i]);
+		if ( r < 0 ) {
+			/* FIXME: perhaps try to update currtermios */
+			term_errno = TERM_ESETATTR;
+			rval = -1;
+			break;
+		}
+#endif /* of __linux__ */
+	} while (0);
+
+	return rval;
+}
+
+/***************************************************************************/
+
+int
+term_lower_rts(int fd)
+{
+	int rval, r, i;
+
+	rval = 0;
+
+	do { /* dummy */
+
+		i = term_find(fd);
+		if ( i < 0 ) { 
+			rval = -1;
+			break;
+		}
+
+#ifdef __linux__
+		{
+			int opins = TIOCM_RTS;
+
+			r = ioctl(fd, TIOCMBIC, &opins);
+			if ( r < 0 ) {
+				term_errno = TERM_ERTSDOWN;
+				rval = -1;
+				break;
+			}
+		}
+#else
```

Comment:
Again, why this else-block for non-Linux?
In the raise/lower DTR case, the portable way to do it is to set the baudrate to 0 (B0). 
I don't think this is true for RTS